Update readme.md (#6)
bs-community/awesome-minecraft: readme.md

- [Planet Minecraft](https://www.planetminecraft.com/forums/) - Creative community fansite.
- [Minecraft Chinese Forum](http://www.mcbbs.net/) - Largest Chinese players community of Minecraft.

## Mod Loaders

- [Minecraft Forge](https://github.com/MinecraftForge/MinecraftForge) - Modifications to the Minecraft base files to assist in compatibility between mods.
- [Fabric](https://fabricmc.net/) - A lightweight, experimental modding toolchain for Minecraft.
- [liteloader](http://www.liteloader.com/) - A lightweight mod loader for Minecraft designed to provide simple, high-performance and reliable loader functionality for mods which don't need to modify game mechanics.
- [Meddle](https://github.com/FyberOptic/Meddle) - Everything related to modding 1.9/1.10 snapshots in one place.
- [RopeMC](https://github.com/RopeMC/RopeMC) - A lightweight, reflection based mod loader which aims to be user friendly and later version independent.

## Forge Mods

### Performance

- [Optifine](https://www.optifine.net/home) - OptiFine is a Minecraft optimization mod.

> - [BetterFps](https://www.curseforge.com/minecraft/mc-mods/betterfps) - This is a mod that does what it says it does.

Reviewer: Copied from CurseForge:

```suggestion
- [BetterFps](https://www.curseforge.com/minecraft/mc-mods/betterfps) - BetterFps is a Minecraft mod that add a few performance improvements.
```

### Items Management

> - [Just Enough Items (JEI)](https://minecraft.curseforge.com/projects/jei) - JEI is an item and recipe viewing mod for Minecraft, built from the ground up for stability and performance. VERY expandable in terms of compatability (just search up "Just Enough" + Mod Name of choice)

Reviewer: Copied from CurseForge:

```suggestion
- [Just Enough Items (JEI)](https://minecraft.curseforge.com/projects/jei) - JEI is an item and recipe viewing mod for Minecraft, built from the ground up for stability and performance.
```

> - [Bibliocraft](https://www.curseforge.com/minecraft/mc-mods/bibliocraft) - "Bibliocraft is a storage and organizational mod that began as a single bookcase and has expanded to so much more."

Reviewer: ```suggestion
- [Bibliocraft](https://www.curseforge.com/minecraft/mc-mods/bibliocraft) - Bibliocraft is a storage and organizational mod that began as a single bookcase and has expanded to so much more.
```

Reviewer: The quotes haven't been removed.

- [Thermal Foundation + Dynamics + Expansion](https://minecraft.curseforge.com/projects/thermalexpansion) - A server-friendly and content-rich blend of magic and technology! Title is three different mod names, but of the same series and usually played together.
- [Ender IO](https://github.com/CrazyPants/EnderIO) - Ender IO is a full-sized tech-mod.
- [Railcraft](https://github.com/Railcraft/Railcraft) - Greatly expand and improve the Minecart system in Minecraft.
- [Actually Additions](https://www.curseforge.com/minecraft/mc-mods/actually-additions) - Actually Additions is a technology mod that adds various utilities like multitools, item magnets, atomic reconstructors and so on.
- [Applied Energistics 2](https://github.com/AppliedEnergistics/Applied-Energistics-2) - A Minecraft Mod about Matter, Energy and using them to conquer the world. Mainly focused on item storage.
- [Draconic Evolution](https://www.curseforge.com/minecraft/mc-mods/draconic-evolution) - One of the best tech mod there is, but really overpowered in the late game. Adds things like a Draconic Power Core which no one has been able to fill yet...

> - [Ender IO](https://www.curseforge.com/minecraft/mc-mods/ender-io) - A large ender themed mod, that just adds lots of new fun things. (IO meaning input/output)

Reviewer: Duplicated, please consider updating existing one.

Reviewer: It seems that this hasn't not resolved yet.

- [Galacticraft](https://micdoodle8.com/mods/galacticraft) - A mod that adds rockets and new planets to explore. Expandable with addons.

> - [ComputerCraft](https://www.computercraft.info) - Adds computers based on LUA (5.2 and 5.3 as of writing) into Minecraft.

Reviewer: ```suggestion
- [OpenComputers](https://ocdoc.cil.li) - OpenComputers is a mod that adds computers and robots into the game, which can be programmed in Lua 5.3.
- [ComputerCraft](https://www.computercraft.info) - Adds computers based on Lua (5.2 and 5.3) into Minecraft.
```

- [WarpDrive](https://www.curseforge.com/minecraft/mc-mods/warpdrive) - Adds Star Trek and other franchise like ships, weapons, transporters and so on into the game. Also adds new planets (dimensions).

### Farming And Food

- [Pam's HarvestCraft](https://minecraft.curseforge.com/projects/pams-harvestcraft) - Pam's HarvestCraft is a mod for Minecraft that currently adds many kinds of crops and food.
- [AgriCraft](https://www.curseforge.com/minecraft/mc-mods/agricraft) - A mod that adds a more sophisticated crop system, like the ability to have genetic mutations and also adds more crops.
- [Bonsai Trees](https://www.curseforge.com/minecraft/mc-mods/bonsai-trees) - Adds essentially mini tree farms which are one block big.

> - [Ex Nihilo: Creatio](https://www.curseforge.com/minecraft/mc-mods/ex-nihilo-creatio) - A great mod for skyblocks, it filters things like sand and gravel for useful resources like iron and diamond, compatible with lots of mods. (Ex Nihilo: Creatio in latin roughly translates to Out of Nothing: I Create)

Reviewer: I think that description is unnecessary...

```suggestion
- [Ex Nihilo: Creatio](https://www.curseforge.com/minecraft/mc-mods/ex-nihilo-creatio) - A great mod for skyblocks, it filters things like sand and gravel for useful resources like iron and diamond, compatible with lots of mods.
```

- [Bonsai Trees](https://www.curseforge.com/minecraft/mc-mods/bonsai-trees) - Adds essentially mini tree farms which are one block big.
- [Ex Nihilo: Creatio](https://www.curseforge.com/minecraft/mc-mods/ex-nihilo-creatio) - A great mod for skyblocks, it filters things like sand and gravel for useful resources like iron and diamond, compatible with lots of mods. (Ex Nihilo: Creatio in latin roughly translates to Out of Nothing: I Create)

### Energy

- [RFTools](https://minecraft.curseforge.com/projects/rftools) - RFTools is a mod containing various tools that are related to RF (Redflux) or technology in general.

> - [Applied Energistics 2](https://github.com/AppliedEnergistics/Applied-Energistics-2) - A Minecraft Mod about Matter, Energy and using them to conquer the world.

Reviewer: Why remove this one?

Author: I think I moved it to Item Management, since, I personally think AE2 is mostly focused on storage, but that's just a personal opinion.

### Magic

- [AbyssalCraft](https://www.curseforge.com/minecraft/mc-mods/abyssalcraft) - AbyssalCraft is a magic mod centered around exploration.
- [Astral Sorcery](https://www.curseforge.com/minecraft/mc-mods/astral-sorcery) - This is a magic mod based on the stars, and harnessing the powers of starlight and the constellations.
- [Avaritia](https://www.curseforge.com/minecraft/mc-mods/avaritia) - Avaritia is a mod with a really long progression tree, but really overpowered in the late game. Its soul purpose is to drag the endgame gameplay to extreme lengths.
- [Blood Magic](https://www.curseforge.com/minecraft/mc-mods/blood-magic) - This is a magic mod based on blood. Beware, you die a lot.

### Utilities

- [TNTUtils](https://github.com/ljfa-ag/TNTUtils) - More control over explosions in Minecraft.

> - [Akashic Tome](https://www.curseforge.com/minecraft/mc-mods/akashic-tome) - A mod based on Morph-o-Tool, it mainly gives the ability to merge different mods' documentation books.

Reviewer: ```suggestion
- [Akashic Tome](https://www.curseforge.com/minecraft/mc-mods/akashic-tome) - A mod based on Morph-o-Tool, which mainly gives the ability to merge different mods' documentation books.
```

- [AppleSkin](https://www.curseforge.com/minecraft/mc-mods/appleskin) - This mod does not change any existing mechanics, but simply makes the existing hunger saturation and exhaustion levels visible in the default GUI.
- [Attribute Fixer](https://www.curseforge.com/minecraft/mc-mods/attributefix) - This mod fixes the attribute system in Minecraft which handles things like armour points, because many other mods are inadvertantly affected by a limit in this system.

> - [Diet Hopper](https://www.curseforge.com/minecraft/mc-mods/diet-hoppers) - "This mod modifies the way Minecraft calculates the collision boxes of the hopper, so that you can access any visible blocks behind it."

Reviewer: ```suggestion
- [Diet Hopper](https://www.curseforge.com/minecraft/mc-mods/diet-hoppers) - This mod modifies the way Minecraft calculates the collision boxes of the hopper, so that you can access any visible blocks behind it.
```
